README.md

```diff
@@ -89,6 +90,56 @@ Example `.eslintrc.json` for a **TypeScript React** project:
 }
 ```
 
+Example `.eslintrc.json` for a **Vue 3** project:
+
+```json
+{
+  "root": true, 
+  "extends": [
+    "hardcore",
+    "hardcore/ts-for-js",
+    "hardcore/vue"
+  ], 
+  "parserOptions": {
+    "parser": "@typescript-eslint/parser", 
+    "project": "tsconfig.json"
+  },
+  "settings": {
+    "import/resolver": {
+      "alias": {
+        "map": [["@", "./src/"]], 
+        "extensions": [".js", ".vue"]
+      }
+    }
+  }
+}
+```
+<sub>Make sure to also add **eslint-import-resolver-alias** to your project.<br>If you would like to use **ts-for-js**, do not forget to create a **tsconfig.json** file containing ```{}```. If you would not like to use it, you can omit the **parserOptions** section. </sub>
+
+Example `.eslintrc.json` for a **Typescript Vue 3/Nuxt 3** project:
+
+```json
+{
+  "root": true, 
+  "extends": [
+    "hardcore",
+    "hardcore/ts",
+    "hardcore/vue"
+  ],
+  "parserOptions": {
+    "parser": "@typescript-eslint/parser",
+    "project": "tsconfig.json"
+  },
+  "settings": {
+    "import/resolver": {
+      "typescript": {}
+    }
+  }
+}
+
+```
+<sub>Make sure to also add **eslint-import-resolver-typescript** to your project.<br>For **Nuxt**: make sure to add ```strict: true``` under **compilerOptions** in your **tsconfig.json**.</sub>
```

Comment:
I think I should try to include eslint-import-resolver-typescript into eslint-config-hardcore, to avoid making users install anything manually.

Reply:
I added eslint-import-resolver-typescript in 01403edb782969003afb0a4118dc722568721b0a. I hope it will work out of the box. Now you can omit the note about eslint-import-resolver-typescript from README.

Reply:
Released in [v24.10.0](https://github.com/EvgenyOrekhov/eslint-config-hardcore/releases/tag/v24.10.0).